mri/operators/fourier/utils.py

```diff
@@ -97,11 +97,11 @@ def normalize_frequency_locations(samples, Kmax=None):
     """
     samples_locations = np.copy(samples.astype('float'))
     if Kmax is None:
-        Kmax = 2*np.abs(samples_locations).max(axis=0)
+        Kmax = np.abs(samples_locations).max(axis=0)
     elif isinstance(Kmax, (float, int)):
         Kmax = [Kmax] * samples_locations.shape[-1]
     Kmax = np.array(Kmax)
-    samples_locations /= Kmax
+    samples_locations /= 2*Kmax
```

Comment:
Why do we need this change? It would be better to place Kmax to actually represent Kmax, ie 0.5 and not 1.
Lets not make changes if its unnecessary..

Reply:
I think we've done this change to fix the problem we had with rotating stacks and the fact it was changing the FOV.

Reply:
How would this solve the issue? The `sample_locations` would still be the same in both the cases, we merely moved when we divide by 2.

Reply:
Because at the begining we `*2` was only applied in the case Kmax is none and we needed to apply it also when we give a specific kmax : 
`elif isinstance(Kmax, (float, int)):	    
        Kmax = 2 * [Kmax] * samples_locations.shape[-1]` 
And instead of multiplying by 2 in both case it was easier to just divide by 2 the final kmax.

Reply:
Hmm interesting. 
Makes sense.